### ISSUE TITLE
muCommander main window doesn't have keyboard focus after switching from another window

### ISSUE THREAD

**Issue by SlavikBaranov** [mucommander/mucommander#347]
Hi! I'm using your great app for many years and after upgrading my mac I'm observing annoying problem.

Steps to reproduce:

- Navigate with arrows to verify keyboard shortcuts work
- Press `F3` to view a file in a separate window and press `Esc` to exit viewer (or open any confirmation dialog or just switch to another application and back with `Cmd+Tab`)
- Try to press arrow keys, `Esc`, etc.
    * Expected behaviour: keyboard shortcuts work
    * Actual behaviour: keyboard shortcuts doesn't work

```
muCommander
Version: 0.9.4
Build date: 2020/03/08
Build number: 8f24dfc

Java
Runtime version: 11.0.4
VM name: OpenJDK 64-Bit Server VM
VM version: 11.0.4+11
VM vendor: AdoptOpenJDK

OS
Name: Mac OS X
Version: 10.15.4
Architecture: x86_64
```

**Comment by ahadas** [comment]
Hi @SlavikBaranov, thanks for reporting this.

> * Press `F3` to view a file in a separate window and press `Esc` to exit viewer (or open any confirmation dialog or just switch to another application and back with `Cmd+Tab`)

I can't reproduce this on my machine with the same version (of muCommander and Java). The only difference is that I'm using macOS 10.15.5. It might be an issue at the operating system <-> JVM  level (I saw [this](https://members.symless.com/forums/topic/6479-keyboard-focus-problems-all-mac-setup/) that may suggest there was an issue in 10.15.4).

Could you try to upgrade to 10.15.5 and check if it still happens?

**Comment by SlavikBaranov** [comment]
Hi @ahadas, thanks for looking into this. 

I couldn't reproduce this after upgrading to 10.15.5, so I guess the issue might be closed. 

Big thanks for making this great app!
